src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -113,12 +114,21 @@ public void save(AddressBook addressBook) throws StorageOperationException {
 
     /**
      * Loads data from this storage file.
-     *
+     * 
+     * @return an {@link AddressBook} containing the data in the file, or an empty {@link AddressBook} if it
+     *    does not exist.
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
      */
     public AddressBook load() throws StorageOperationException {
+        File fileOnDisk = path.toFile();
+
+        if (!fileOnDisk.exists()) {
+            final AddressBook empty = new AddressBook();
```

Comment:
Change the name from `empty` to `emptyAddressBook`. Otherwise, just delete this line and `return new AddressBook()` since it seems rather intuitive that `new AddressBook()` is an empty one